pinot-controller/src/test/java/com/linkedin/pinot/controller/validation/StorageQuotaCheckerTest.java

```diff
@@ -51,8 +53,10 @@ public void setUp() {
     _quotaConfig = mock(QuotaConfig.class);
     _controllerMetrics = mock(ControllerMetrics.class);
     _validationConfig = mock(SegmentsValidationAndRetentionConfig.class);
+    _pinotHelixResourceManager = mock(PinotHelixResourceManager.class);
     when(_tableConfig.getValidationConfig()).thenReturn(_validationConfig);
     when(_validationConfig.getReplicationNumber()).thenReturn(2);
+    when(_pinotHelixResourceManager.isLeader()).thenReturn(true);
```

Comment:
Can you also add a test to check that the emitted % is correct?

Reply:
Test added in the latest push.